api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -68,10 +70,21 @@
 
     private MessageBirdService messageBirdService;
 
+
+    public enum Feature {
+        ENABLE_CONVERSATION_API_WHATSAPP_SANDBOX
+    }
+
     public MessageBirdClient(final MessageBirdService messageBirdService) {
         this.messageBirdService = messageBirdService;
     }
 
+    public MessageBirdClient(final MessageBirdService messageBirdService, Feature[] features) {
```

Comment:
It's better practice to use a `List` not a native array.

Reply:
Changed to list